library/catalog/templates/book-details.html

```diff
@@ -14,10 +14,12 @@
 
 {% block body %}
 
-    <div style="margin-top: 100px;background-color: #e1e5e7;" class="container border p-3 mb-3 book_details_frame rounded">
+    <div style="margin-top: 100px;background-color: #e1e5e7;"
```

Comment:
It will be good to move inline CSS to external file

Reply:
@ivan-dubei  Think we'll do it later. wasn't my part of work